src/ClientModel.h

```diff
@@ -150,9 +152,9 @@ class RecordLogEntry: public QObject
 	RecordLogEntry():
 		m_recordIndex(0), m_call(false), m_type(RecordType::Transaction) {}
 	RecordLogEntry(unsigned _recordIndex, QString _transactionIndex, QString _contract, QString _function, QString _value, QString _address, QString _returned, bool _call, RecordType _type, QString _gasUsed,
```

Comment:
outOfGas is not the only possible error. There are a few more, like invalid insruction, bad jump destination, stack error, etc. Would be nice to handle them as well